openadapt/ml/prompt_completion.json

```diff
@@ -0,0 +1,356 @@
+{"prompt": "({'name': 'move', 'mouse_x': 949.84765625, 'mouse_y': 629.609375, 'element_state': {}}, {'title': 'Terminal ml \u2014 poetry shell \u25b8 Python \u2014 204\u00d742', 'left': 2, 'top': 86, 'width': 1438, 'height': 623, 'window_id': 169139})", "completion": "({'name': 'move', 'mouse_x': 949.23046875, 'mouse_y': 628.68359375, 'element_state': {}}, {'title': 'Terminal ml \u2014 poetry shell \u25b8 Python \u2014 204\u00d742', 'left': 2, 'top': 86, 'width': 1438, 'height': 623, 'window_id': 169139})"}
```

Comment:
The current strategy, (i.e for any timestamp _k_, we have the prompt as the _kth_ (Action, Window) and the completion as the _k+1_th (Action, Window) pair) can be seen here.